.github/workflows/build-envoy-images-release.yaml

```diff
@@ -79,7 +79,7 @@ jobs:
             BAZEL_BUILD_OPTS="--jobs=HOST_CPUS*.75"
             BAZEL_TEST_OPTS=--test_timeout=300 --local_test_jobs=1
           push: true
-          tags: quay.io/${{ github.repository_owner }}/cilium-envoy-builder:test-master-archive-latest
+          tags: quay.io/${{ github.repository_owner }}/cilium-envoy-builder:test-main-archive-latest
```

Comment:
I think we can just merge this PR as the main-archive-latest image will only build in main branch.